packages/oncokb-frontend-commons/src/components/Cache.tsx

```diff
@@ -0,0 +1,8 @@
+export interface ICacheData<T = any> {
+    status: 'pending' | 'complete' | 'error';
+    data?: T;
+}
+
+export interface ICache<T = any> {
+    [queryId: string]: ICacheData<T>;
+}
```

Comment:
Do we use these types anywhere anymore? We should remove this file if not used anymore.